ui/src/app/images/views/ImageList/CustomImages/CustomImages.tsx

```diff
@@ -0,0 +1,46 @@
+import { Strip } from "@canonical/react-components";
+import { Formik } from "formik";
+import { useSelector } from "react-redux";
+
+import ImagesTable from "app/images/components/ImagesTable";
+import type { ImageValue } from "app/images/types";
+import bootResourceSelectors from "app/store/bootresource/selectors";
+import { splitResourceName } from "app/store/bootresource/utils";
+
+const CustomImages = (): JSX.Element | null => {
+  const uploadedResources = useSelector(
+    bootResourceSelectors.uploadedResources
+  );
+
+  if (uploadedResources.length === 0) {
+    return null;
+  }
+
+  const images = uploadedResources.reduce<ImageValue[]>((images, resource) => {
+    const { os, release } = splitResourceName(resource.name);
+    images.push({
+      arch: resource.arch,
+      os,
+      release,
+      title: resource.title,
+    });
+    return images;
+  }, []);
+
+  return (
+    <>
+      <hr />
+      <Strip shallow>
+        <h4>Custom images</h4>
+        {/* ImagesTable requires formik context in order to be used in conjunction
+          with the image selection checkboxes for synced resources,
+          so we wrap it in Formik despite these resources being uploaded */}
```

Comment:
I think maybe instead we should split up ImagesTable into two components, one that has the table and accepts an array of images and another array of unchecked resources (maybe just the ids) and a second component that contains the formik parts which would wrap the first component. Then CustomImages can just use the first component.

Reply:
I could probably just make `images` a prop instead of using `useFormikContext` in the component. Think that would work?

Reply:
Yeah I think so.

Reply:
@huwshimi ok should be ready for another look now. I've moved `images` to be a `prop` and also removed the `onSuccess` from `DeleteImageConfirm`. Turns out just using the Formik `enableReinitialize` prop achieves the same thing (removing an image from the form values if it no longer exists)